### ISSUE TITLE
[BUG] The new useAnimate sequence feature causes errors and warnings

### ISSUE THREAD

**Issue by sukjae** [framer/motion#2059]
**1. Describe the bug**

Hello. 
In the example below, I expected sequence 1, 2, and 3 to all work the same.
However, some errors and warnings are coming out. 

### sequence error
An error is coming out when using sequence. 
Am I misunderstanding the sequence? 

The sequence that I understood is the same concept as timeline, and I thought it was similar to timeline in gsap and sequence in motion one. 

However, if you run multiple motions in series in the sequence one example, the following error occurs. 

```js
animate([
  [".one", { x: 100 }],
  [".one", { y: 100 }]
]);
```

> Error
All keyframes must be of the same type

<img width="525" alt="image" src="https://user-images.githubusercontent.com/8156543/229001614-ad7f43b0-c29e-4c60-84e1-3912d17ad049.png">

### transform warning
When performing transform sequentially, the following warning occurs. 
The app doesn't shut down, but it doesn't do anything. 

```js
await animate([".three", { transform: "translateX(100px)" }]);
await animate([".three", { transform: "translateY(100px)" }]);
```

> You are trying to animate  from ".three,[object Object]" to "undefined". .three,[object Object] is not an animatable value - to enable this animation set .three,[object Object] to a value animatable to undefined via the `style` property.

<img width="1453" alt="image" src="https://user-images.githubusercontent.com/8156543/229001643-3d74ddda-76a2-46cb-bf8a-d8d928dd7ab4.png">

**2. IMPORTANT: Provide a CodeSandbox reproduction of the bug**

- https://codesandbox.io/s/great-cherry-pxgogh?file=/src/App.js

**3. Steps to reproduce**

- click play buttons in codesandbox.

**Comment by sukjae** [comment]
The second problem was not an error. It was an issue that had to be wrapped up in another arrangement.

### as-is

```js
await animate([".three", { transform: "translateX(100px)" }]);
await animate([".three", { transform: "translateY(100px)" }]);
```

### to-be

```js
await animate([[".three", { transform: "translateX(100px)" }]]);
await animate([[".three", { transform: "translateY(100px)" }]]);
```

**Comment by mattgperry** [comment]
The first issue will be fixed in `10.11.0` going out now.

```javascript
await animate([[".three", { transform: "translateX(100px)" }]]);
```

Can just be

```javascript
await animate(".three", { transform: "translateX(100px)" });
```

And it's probably true that as the `transform` has been read from the DOM it is being read as `"none"`. I think this can be made nicer but you can fix now by providing your own initial keyframe.

```
await animate(".three", { transform: ["translateX(0)", "translateX(100px)"] });
```
